From f12f243e39c840b1306bf376c49ef4dab9e7564f Mon Sep 17 00:00:00 2001
From: lazedo <luis.azedo@factorlusitano.com>
Date: Tue, 24 Feb 2015 17:19:15 +0000
Subject: [PATCH 2/2] presence_dialoginfo: rfc4235 in dummy dialog

according to rfc4235 the id element is mandatory.
granstream phones are affected by this.
---
 modules/presence_dialoginfo/notify_body.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/presence_dialoginfo/notify_body.c b/modules/presence_dialoginfo/notify_body.c
index bd2097b..508b9f8 100644
--- a/modules/presence_dialoginfo/notify_body.c
+++ b/modules/presence_dialoginfo/notify_body.c
@@ -68,7 +68,7 @@ void free_xml_body(char* body)
 }
 
 #define DIALOGINFO_EMPTY_BODY "<dialog-info>\
-<dialog direction=\"recipient\">\
+<dialog id=\"615293b33c62dec073e05d9421e9f48b\" direction=\"recipient\">\
 <state>terminated</state>\
 </dialog>\
 </dialog-info>"
-- 
2.1.4

